modules/weko-search-ui/weko_search_ui/utils.py

```diff
@@ -246,36 +248,43 @@ def get_journal_info(index_id=0):
 
 
 def get_feedback_mail_list():
-    """Get tree items."""
+    """Get feedback items."""
     records_search = RecordsSearch()
     records_search = records_search.with_preference_param().params(
         version=False)
     records_search._index[0] = current_app.config['SEARCH_UI_SEARCH_INDEX']
-    search_instance = feedback_email_search_factory(None, records_search)
-    search_result = search_instance.execute()
-    rd = search_result.to_dict()
-    return rd.get('aggregations').get('feedback_mail_list')\
-        .get('email_list').get('buckets')
+    ret = {}
+
+    try:
+        search_instance = feedback_email_search_factory(None, records_search)
+        aggr = search_instance.execute().to_dict()\
+            .get('aggregations', {})\
+            .get('feedback_mail_list', {})\
+            .get('email_list', {})\
+            .get('buckets', [])
+    except NotFoundError:
+        current_app.logger.debug('FeedbackMail data cannot found!')
+        return ret
+    except InvalidQueryRESTError:
```

Comment:
pls join the exceptions to single line:
Ex: except (NotFoundError, InvalidQueryRESTError):